packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.ts

```diff
@@ -517,31 +527,31 @@ function structureOAuthMetadata(coreAnswers, context, defaults, amplify) {
   }
 
   return coreAnswers;
-}
+};
 
 /*
   Deserialize oAuthData for CLI update flow
 */
-function parseOAuthMetaData(previousAnswers) {
+const parseOAuthMetaData = (previousAnswers: any): any => {
```

Comment:
is this return type void?

Reply:
indeed it is. Changing.

Reply:
I went through `auth-questions.ts` and added more descriptive return types where possible.